docroot/themes/custom/uids_base/templates/layout/footer.html.twig

```diff
@@ -17,6 +17,7 @@
         logo_classes: 'logo-icon--tab',
         logo_path_2: uiowa_logo_path,
         logo_path_png: uiowa_logo_path_png,
+        logo_id: 'uiowa-wordmark-footer',
```

Comment:
Could we change this to `logo-footer`? Getting away from the `uiowa-` naming scheme. This is consistent with the UIDS PR.